src/components/FetchHandler.tsx

```diff
@@ -27,28 +25,27 @@ export const FetchHandler = <TResult extends {}>({
     notFoundIndicator,
     loadingIndicator,
 }: Props<TResult>) => {
-    const { loading, data, error, trigger, errorCode } = usePromise(apiMethod);
-    const [dependencyList, setDependencyList] = useState(undefined);
+    const { loading, data, error, trigger } = usePromise(apiMethod);
 
     useEffect(() => {
         trigger();
-    }, [dependencyList, trigger]);
+    }, [trigger]);
 
     const renderError = () => {
-        if (errorIndicator && (errorCode !== 404 && notFoundIndicator)) {
+        if (errorIndicator && error?.code !== 404 && notFoundIndicator) {
             return errorIndicator;
-        } else if (errorCode === 404 && notFoundIndicator) {
+        } else if (error?.code === 404 && notFoundIndicator) {
             return notFoundIndicator;
         }
 
-        return errorCode !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
+        return error?.code !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
     };
```

Comment:
Finns det inte fler error koder så som 422 eller liknande som kan vara bra att ta tag i och isåfall kunna möjliggöra hantering?

Reply:
Translate : "Are there no more error codes such as 422 or the like that can be good to grasp and, in that case, be able to allow handling a little more specifically?"

Reply:
It's out of scope for this fix.